apps/backend-functions/src/internals/invitations/events.ts

```diff
@@ -196,6 +197,25 @@ export async function createNotificationsForEventsToStart() {
   return notifications.length ? triggerNotifications(notifications) : undefined;
 }
 
+export async function createNotificationsForFinishedScreenings() {
+  const halfAnHour = 1800 * 1000;
+  const screenings = await fetchFinishedScreenings(halfAnHour); 
+  const notifications = [];
+
+  for (const screening of screenings) {
+    const attendees = screening.meta.attendees ? Object.keys(screening.meta.attendees) : [];
+    const invitations = await fetchEventInvitations(screening.id);
+
+    for (const invitation of invitations) {
+      const userId = getGuest(invitation, 'user').uid;
+      const notificationType = attendees.includes(userId) ? 'postScreening' : 'userMissedScreening';
+      const notification = await createNotificationIfNotExists([invitation], notificationType);
+      notifications.push(notification[0]);
```

Comment:
Actually this might be even nicer :)
```suggestion
      const [notification] = await createNotificationIfNotExists([invitation], notificationType);
      notifications.push(notification);
```